e2e/test/scenarios/question/notebook.cy.spec.js

```diff
@@ -297,6 +297,9 @@ describe("scenarios > question > notebook", { tags: "@slow" }, () => {
 
   describe("arithmetic (metabase#13175)", () => {
     beforeEach(() => {
+      // This is required because TableInteractive won't render columns
+      // that don't fit into the viewport
+      cy.viewport(1400, 1000);
```

Comment:
@oleggromov 

For the future reference, please use Cypress' inline test config for this.
```
  describe(
    "arithmetic (metabase#13175)",
    {
      viewportWidth: 1400,
      viewportHeight: 1000,
    },
    () => {
      beforeEach(() => {
```

That way Cypress already starts the test with that viewport size, so there's no readjusting when the test is already underway. For this particular example it shouldn't make any difference because we're resizing even before we visit any page so effectively it's the same. But there might be some cases where doing this in the middle of the test introduces more trouble.

Reply:
Thanks, noted. I am not doing it in the middle of a suite though.

Reply:
I know. That's why I'm saying it doesn't make a difference in this particular example.

Sometimes when we have nested describe blocks, people unknowingly run into this trap. Some of the previous blocks might have a step that's already using UI.

Again, not the case here.